src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyTotal = 0;
+        int buyTotal = 0;
+        int supplyCount = 0;
+        int buyCount = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                 BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
```

Comment:
nested connections problem as described [here ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#dont-open-nested-connection-to-files)